src/main.cpp

```diff
@@ -690,7 +690,15 @@ void GetGlobalStatus()
         GlobalStatusStruct.ERRperday = RoundToString(boincmagnitude * GRCMagnitudeUnit(GetAdjustedTime()),2);
         GlobalStatusStruct.project = msMiningProject;
         GlobalStatusStruct.cpid = GlobalCPUMiningCPID.cpid;
-        GlobalStatusStruct.poll = msPoll;
+        try
+        {
+            GlobalStatusStruct.poll = GetCurrentOverviewTabPoll();
+        }
+        catch (std::exception &e)
+        {
+            GlobalStatusStruct.poll = "Error obtaining last poll";
```

Comment:
Translations should be considered here. 
`GlobalStatusStruct.poll = _("Error obtaining last poll");`

Reply:
I agree if anything a log entry about the exception should be put. Remove translation and log the e.what() under debug